packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -68,7 +68,8 @@ def test_octarine_http(cluster, timeout=30):
     octarine_id = uuid.uuid4().hex
     proxy = ('"http://127.0.0.1:$(/opt/mesosphere/bin/octarine ' +
              '--client --port {})"'.format(octarine_id))
-    check_command = 'curl --fail --proxy {} marathon.mesos'.format(proxy)
+    url = "marathon.mesos.mydcos.directory"
```

Comment:
given this is a locally used only thing, why not just embed it as a constant in the check_command string?